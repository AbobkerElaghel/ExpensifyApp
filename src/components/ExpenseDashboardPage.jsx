import React from "react";
import ExpenseList from "./ExpenseList.jsx"
import ExpenseListFilter from "./ExpenseListFilter.jsx";
import ExpenseSummary from "./ExpenseSummary.jsx";

const ExpenseDashboardPage = () => (
    <div>
        <ExpenseSummary/>
        <ExpenseListFilter/>
        <ExpenseList/>
    </div>
);

export default ExpenseDashboardPage;