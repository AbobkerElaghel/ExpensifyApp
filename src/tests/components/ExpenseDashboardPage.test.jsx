import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage.jsx";

test('should render ExpenseForm correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});