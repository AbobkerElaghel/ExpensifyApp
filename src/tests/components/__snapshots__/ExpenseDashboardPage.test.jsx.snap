// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseForm correctly 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
